docs/source/user-guide/concepts/installing-with-conda.rst

```diff
@@ -2,41 +2,43 @@
 Installing with conda
 =====================
 
-.. image:: /img/installing-with-conda.png
-    :align: right
-
 .. _installing-with-conda:
 
+Conda packages can be installed by running the following command::
 
-To install conda packages, in the terminal or an Anaconda Prompt, run::
-
-  conda install [packagename]
+  conda install <package>
```

Comment:
We should attempt to have a standard for placeholders in commands going forward. I'm not married to any one thing, so `<all-lowercase-words>` works for me. I've also used all-caps, e.g. `<PACKAGE>`.

Any opinions?

Reply:
I like `<all-lowercase>`. I always get a little anxious when I SEE ALL CAPS!!! 🙀.

Reply:
🧢🧢🧢

Reply:
Works for me!